microcosm_flask/conventions/crud.py

```diff
@@ -320,9 +319,7 @@ def configure_createcollection(self, ns, definition):
         @wraps(definition.func)
         def create_collection(**path_data):
             request_data = load_request_data(definition.request_schema)
-            # NB: if we don't filter the request body through an explicit page schema,
-            # we will leak other request arguments into the pagination query strings
-            page = self.page_cls.from_query_string(self.page_schema(), request_data)
+            page = self.page_cls.from_query_string(self.page_schema(), {})
```

Comment:
This was a tad confusing to me. I removed this because it started throwing up validation errors because it was trying to load `request_data` fields as query fields, which it was failing at.

I'm probably misunderstanding why we'd want "page" in the first place.

Reply:
so it looks like this is trying to support query params in batch create operations. In theory this seems like it could be useful if the client somehow posts elements from a batch in pages/or if we use query params to control things like inferring fields of nested child schemas from the parent. I'm not sure the former is a real thing. I know we do the latter, but I'm not sure if we control the behavior with query params.

If you are going to run the `page_schema` against and empty dict, you may as well not run this line at all as it will never get data.

Reply:
Right. I tried taking a look at the commit that introduced this, but didn't see anything interesting here. There may be a commit with the exact reasoning.

Reply:
I think that maybe Sphinx uses that ability to pass params in batch operations.

Reply:
I'll double-check this.